docs/how-it-works.md

```diff
@@ -0,0 +1,20 @@
+## How Harden-Runner Works?
+
+### GitHub-Hosted Runners
+
+For GitHub-hosted runners, Harden-Runner GitHub Action downloads and installs the StepSecurity Agent.
+
+- The code to monitor file, process, and network activity is in the Agent.
+- The agent is written in Go and is open source at https://github.com/step-security/agent
```

Comment:
Only the community tier agent is open-source. Our enterprise agent for GitHub-Hosted Runners is closed source.